src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -1,81 +1,343 @@
 // WE ARE CURRENTLY TRYING OUT THE SingleBookingComponent.js PLEASE REFER TO THAT COMPONENT FOR BOOKING FOR NOW
-
 import React from 'react';
 import { connect } from 'react-redux';
-import { Card, Button } from 'antd';
+import { Typography, Input, Select, Layout, Form, Calendar } from 'antd';
 import { dateConverter } from '../../common/dateHelpers';
 import { timeConverter } from '../../common/timeHelpers';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
 import { useOktaAuth } from '@okta/okta-react';
 import { addToCart } from '../../../redux/actions/parentActions';
+import { dummyData } from '../../../dummyData';
```

Comment:
Good job labeling your data as such to indicate it will need to change.

Reply:
I'd go one step further here and indicate. 'what kind of dummyData' it is. Example 

dummyUserData etc.